malaria24/ona/tests/base.py

```diff
@@ -14,6 +16,8 @@
 @override_settings(CELERY_ALWAYS_EAGER=True)
 class MalariaTestCase(TestCase):
 
+    posted = models.DateTimeField()
```

Comment:
I don't think that we're using this variable anywhere else? If not, then we can remove it?